docs/app/documentation/component-docs/combobox/examples/combobox-height-example.component.html

```diff
@@ -0,0 +1,7 @@
+<fd-combobox [(ngModel)]="searchTermOne"
+             [dropdownValues]="dropdownValues"
+             [placeholder]="'Type some text...'"
+             [maxHeight]="'400px'">
```

Comment:
Maybe change it to 250px. Just to let it be scrollable.

Reply:
She added more items in the list to achieve the scroll experience.